pandas/tests/generic/test_generic.py

```diff
@@ -1008,3 +1008,20 @@ def test_pipe_panel(self):
 
             with pytest.raises(ValueError):
                 result = wp.pipe((f, 'y'), x=1, y=1)
+
```

Comment:
this needs to go in pandas/tests/series/test_missing.py

also some tests are failing